src/FritzBox/Converter.php

```diff
@@ -176,6 +191,22 @@ private function getPhoneNumbers()
                 }
             }
         }
+        if (count($phoneNumbers) > 1) {
+            $ordering = $this->phoneSort;
```

Comment:
lass die ordering variable weg

Reply:
Ne - dann geht es auf die Bretter !
VS Code schmeißt 'nen Fehler und im Test schmierts ab.
Ich weiß nicht warum...

Reply:
> VS Code schmeißt 'nen Fehler und im Test schmierts ab.

geht das spezifischer?

Reply:
Sorry, habe dein reply erst nach dem commit & push gesehen :(
Screenshot angekommen?

Reply:
nö, nix gekommen. ich bin 102,5% sicher dass das geht. ohne use() da $this immer erreichbar!

Reply:
Hab's!
Manchmal sieht man den Wald vor lauter Bäumen nicht.
Aber Gogle und stackoverflow ... https://stackoverflow.com/questions/44482168/how-to-use-this-in-closure-in-php